assets/tailwind.config.js

```diff
@@ -100,6 +100,9 @@ module.exports = {
           800: "#854D0E",
           900: "#713F12",
         },
+        "brand-pink": {
+          500: "#e44c75",
+        },
```

Comment:
Here we have just a single variant :D

```suggestion
        "brand-pink": "#e44c75",
```